packages/kit/src/Form.tsx

```diff
@@ -1,91 +0,0 @@
-import { ComponentProps, createContext, ReactNode, useContext, useEffect, useState } from 'react'
```

Comment:
Form doesn't need any special components now. We support it out of the box and can consider adding it to the kit if manual impl. as a part of example turns out to be too complex.